app/views/layouts/application_mailer.html.erb

```diff
@@ -8,4 +8,7 @@
   <body>
     <%= yield %>
   </body>
+  <footer style="display: block; font-size: .75em; font-weight: bold">
+    <%= email_footer %>
```

Comment:
Why a difference between yield for head and body but not the footer?

Reply:
This comment still seems open. The `yield` is for rendering the additional content passed to this layout (e.g. if some mail has extra content for head or in this case for footer). At the same time, we want to always have this footer information in every mail using this layout, that's why also the `<%= email_footer %>`

Perhaps the code has meanwhile changed and the `yield :footer` was added afterwards, given it's a new line, it's not causing this comment being outdated.

@ekohl is this addressed now?

Reply:
When I made the suggestion, `yield :footer` wasn't there. Marking as resolved now.